ogr/services/pagure/project.py

```diff
@@ -405,6 +405,10 @@ def get_file_content(self, path: str, ref=None) -> str:
 
         if not result or result.reason == "NOT FOUND":
             raise FileNotFoundError(f"File '{path}' on {ref} not found")
+        if result.reason != "OK":
+            raise PagureAPIException(
+                f"File '{path}' on {ref} not found due to {result.reason}"
+            )
```

Comment:
Just to check, are we sure that the received `500` errors have the `reason != "OK"`?

Reply:
Sentry shows `reason: INTERNAL SERVER ERROR`

Reply:
Thanks for checking!